backend/schema/corpora_schema.md

```diff
@@ -1,125 +1,68 @@
-
-
-### Submitter Incentives
-
-We recognize that Corpora cannot _make_ submitters do anything; following our schema is voluntary. But, Corpora can create incentives. For example, if a
-submitter doesn't want to include "tissue" then their dataset would not show up in tissue-related search results in the Portal. Or if a submitter uses a
-gene annotation that is incompatible with the Corpora schema annotation, then gene card information will not be available for their dataset in cellxgene.
-
-Over time we expect these incentives will lead to broader adoption of the Corpora schema and less friction around creating the remix datasets.
-
-
-### Software vs Policy
-
-Finally, we should draw a distinction between how we approach the schema and validation as a _policy_ versus how we implement it in _software_. The schema,
-especially anything outside of fundamental requirements, will change over time. It might change frequently and significantly as we get feedback from submitters
-and users. So while we might be strict about our validation policies at any given time, our software needs to be very flexible. We need to avoid a situation
-where we can't evolve schema and curation policies because it requires expensive software updates. There will be situations where software features or
-capabilities will be hard or impossible because we prioritize this flexibility.
+Then, when the submitter uploads the dataset to the Data Portal, the Data Portal will verify that the dataset does indeed follow an accepted version of the
+schema. If it does not, it will reject the dataset with an appropriate error message.
 
 ## Schema
 
-### Original Dataset Requirements
-
-
-These are the fundamental requirements that all dataset must meet, the handful of things our software depends on and mandatory legal requirements. All
-submitted data must comply with these requirements.
+### Basic Requirements
 
+There are a few requirements that are needed for the cellxgene Explorer to work correctly or are mandatory legal requirements:
 
 *   **Unique observation identifiers**. Each observation (usually a cell) must have an id that is unique within the dataset. This is usually already
     present in every submission, for example as a barcode.
 *   **Unique feature identifiers**. Every feature (usually a gene or transcript) also needs a unique identifier. This is occasionally not present because
-    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will be summed during
-    curation.
-*   **Type annotations or inferrable types**. For both expression values and metadata, we don't want to have to guess at the type. Typing is already explicit
-    in anndata, loom, Seurat and Bioconductor formats.
+    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will need to be appropriately
+    combined before submission.
```

Comment:
What does _appropriately **combined**_ mean?

Reply:
I'll add a little sentence of explanation.